### PR TITLE
Explain how to import in 3.8 a template from 3.7 with software bundles

### PR DESCRIPTION
**Pull request by AliceAlbano**

### CODE REVIEW

**Comment by yanngit** [comment]
For me, the "Compatibility issues' should be moved to fp1 section in change log ...

**Comment by AliceAlbano** [comment]
I'm ok with the modifications

File: changelog.rst [hunk 1]
```diff
@@ -32,6 +23,9 @@ Known issues
 ~~~~~~~~~~~~
 * In some situation, deployment in CloudStack could fail if the CloudStack image is duplicated in different zones.
 
+Compatibility issues
+~~~~~~~~~~~~~~~~~~~~
+* The import / export of appliance templates to UForge 3.7 to UForge 3.8 may not work if the template contains software bundles. Please refer to this section :ref:`appliance-update-37-import` to make your template compatible.
```
Comment (alefebvr) [inline changelog.rst]: Replace "to UForge 3.7 to UForge 3.8" with "from UForge 3.7 to UForge 3.8".
Replace "may not work" with "may require additional work"

File: end-user/en/source/pages/templating/appliance-import-export.rst [hunk 2]
```diff
@@ -66,7 +66,7 @@ To import an archive:
 Updating a Windows Appliance Before Import
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When you import a Windows appliance, UForge checks if the golden image used to create the appliance exists on the UForge. If it does not exist, the import will fail. In order to import the template, you must export the appliance and remove the OS profile. 
+When you import a Windows appliance, UForge checks if the golden image used to create the appliance exists on the UForge. If it does not exist, the import will fail. In order to import the template, you must export the appliance and remove the OS profile.
```
Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: Replace "exists on the UForge." with "exists on the UForge server."

File: end-user/en/source/pages/templating/appliance-import-export.rst [hunk 3]
```diff
@@ -124,3 +124,35 @@ For exemple, if your ``oses`` subsection is like this::
 Then you need to write this restriction rule instead::
 
     (Distribution#name=CentOS && Distribution#version=7 && Distribution#arch=x86_64) || (Distributioon#name=Debian && Distribution#version=8 && Distribution#arch=x86_64)
+
+When you export a Windows appliance in UForge 3.7, you will also encounter compatibility issues. In order to import the template, you must modify the file containing the description of your appliance (``template.yml`` or ``template.json``).
```
Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: Note: the export itself does not encounter compability issues, it's the import.
Suggestion: replace "you will also encounter compatibility issues" with "you will also encounter compatibility issues when importing the template into UForge 3.8"
Replace: "In order to import the template" with "In order to be able to import the template into UForge 3.8"

File: end-user/en/source/pages/templating/appliance-import-export.rst [hunk 4]
```diff
@@ -124,3 +124,35 @@ For exemple, if your ``oses`` subsection is like this::
...
+
+In this file, there is a section ``os`` that contains various information and in particular the profile name. The new 3.8 template is now having additional fields, ``windowsEdition``, ``windowsType`` and ``windowsLanguage`` that you need to fulfill to be able to import the template.
```
Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: English rephrasing:
Replace "The new 3.8 template is now having additional fields" with "The 3.8 template file now has additional fields"
Replace "that you need to fulfill" with "that you need to complete"

File: end-user/en/source/pages/templating/appliance-import-export.rst [hunk 5]
```diff
@@ -124,3 +124,35 @@ For exemple, if your ``oses`` subsection is like this::
...
+			name: "Windows"
+			version: "Server2012R2"
+			arch: "x86_64"
+			updateTo: "12-19-2016 10:32:07 GMT"
+			profile: "Standard Full English"
+			windowsEdition: "standard"
+			windowsType: "full"
+			windowsLanguage: "English"
+
+The profile should be existing on the target UForge, otherwise you must delete the ``profile`` field and import the template like this::
```
Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: Replace:
- "The profile should be existing" with "The profile should exist"
- "on the target UForge" with "on the target UForge platform"

File: end-user/en/source/pages/templating/appliance-import-export.rst [hunk 6]
```diff
@@ -124,3 +124,35 @@ For exemple, if your ``oses`` subsection is like this::
 Then you need to write this restriction rule instead::
 
     (Distribution#name=CentOS && Distribution#version=7 && Distribution#arch=x86_64) || (Distributioon#name=Debian && Distribution#version=8 && Distribution#arch=x86_64)
+
+When you export a Windows appliance in UForge 3.7, you will also encounter compatibility issues. In order to import the template, you must modify the file containing the description of your appliance (``template.yml`` or ``template.json``).
+
+In this file, there is a section ``os`` that contains various information and in particular the profile name. The new 3.8 template is now having additional fields, ``windowsEdition``, ``windowsType`` and ``windowsLanguage`` that you need to fulfill to be able to import the template.
+
+For exemple, if your ``os`` section is like this::
```
Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: Replace "For exemple" with "For example"

Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: Replace "For exemple" with "For example"

File: end-user/en/source/pages/templating/appliance-import-export.rst [hunk 7]
```diff
@@ -124,3 +124,35 @@ For exemple, if your ``oses`` subsection is like this::
...
+			windowsLanguage: "English"
+
+The profile should be existing on the target UForge, otherwise you must delete the ``profile`` field and import the template like this::
+
+			name: "Windows"
+			version: "Server2012R2"
+			arch: "x86_64"
+			updateTo: "12-19-2016 10:32:07 GMT"
+
+Thus, your appliance will be imported and you will be able to setup manually in the UI the OSProfile.
```
Comment (alefebvr) [inline end-user/en/source/pages/templating/appliance-import-export.rst]: Replace "you will be able to setup manually in the UI the OSProfile" with "you will be able to setup the OSProfile manually in the UI"

File: changelog.rst (hunk 1 above)
Comment (yanngit) [reply changelog.rst]: done

File: end-user/en/source/pages/templating/appliance-import-export.rst (hunk 2 above)
Comment (yanngit) [reply end-user/en/source/pages/templating/appliance-import-export.rst]: done

File: end-user/en/source/pages/templating/appliance-import-export.rst (hunk 6 above)
Comment (yanngit) [reply end-user/en/source/pages/templating/appliance-import-export.rst]: done

Comment (yanngit) [reply end-user/en/source/pages/templating/appliance-import-export.rst]: done

File: end-user/en/source/pages/templating/appliance-import-export.rst (hunk 5 above)
Comment (yanngit) [reply end-user/en/source/pages/templating/appliance-import-export.rst]: done

File: end-user/en/source/pages/templating/appliance-import-export.rst (hunk 7 above)
Comment (yanngit) [reply end-user/en/source/pages/templating/appliance-import-export.rst]: done

**Comment by AliceAlbano** [comment]
Thanks for all the commits and comments ! This PR is closed to be replaced by PR #598. The content is the same, but it will be merged into branch 3.8.fp1.

**Pull request closed without merging**